components/ui/checkbox.tsx

```diff
@@ -43,7 +31,7 @@ const Checkbox = ({
       <div className="flex h-5 items-center">
         <input
           checked={checked}
-          className={checkboxClasses({ disabled, className })}
+          className={checkboxClasses({ className })}
```

Comment:
Decided to use `disabled` selector for styling when checkbox is disabled as it's cleaner and concise.